DEPENDENCY-LICENSES

```diff
@@ -14,6 +14,7 @@ List of third-party dependencies grouped by their license type.
 
     Apache License
 
+        * carbonite (org.clojars.bipinprasad:carbonite:1.6.0 - https://github.com/bipinprasad/carbonite)
```

Comment:
Wonder, if we should move it into a an ASF dependency, ie. host it as apache/storm-carbonite? This way every committer would be able to patch, etc. ?

Reply:
In the Jira [STORM-3912](https://issues.apache.org/jira/browse/STORM-3912) I outlined the steps to do so. So someone else more familiar and efficient can do it. However, at some point we may have to vote to remove Clojure support in Storm and hence also this dependency.